client/scripts/ui.js

```diff
@@ -242,6 +242,7 @@ class ReceiveDialog extends Dialog {
         this._busy = true;
         const file = this._filesQueue.shift();
         this._displayFile(file);
+        this._saveAutoDownload();
```

Comment:
Here...

Reply:
Yep it's right, it should be this._storeAutoDownload()